library/common/config/config.cc

```diff
@@ -0,0 +1,387 @@
+// NOLINT(namespace-envoy)
+#include "library/common/config/internal.h"
+#include "library/common/config/templates.h"
+
+const char* platform_filter_template = R"(
+        - name: envoy.filters.http.platform_bridge
+          typed_config:
+            "@type": type.googleapis.com/envoymobile.extensions.filters.http.platform_bridge.PlatformBridge
+            platform_filter_name: {{ platform_filter_name }}
+)";
+
+const char* native_filter_template = R"(
+        - name: {{ native_filter_name }}
+          typed_config: {{ native_filter_typed_config }}
+)";
+
+const char* route_cache_reset_filter_insert = R"(
+        - name: envoy.filters.http.route_cache_reset
+          typed_config:
+            "@type": type.googleapis.com/envoymobile.extensions.filters.http.route_cache_reset.RouteCacheReset
+)";
+
+const int custom_cluster_indent = 2;
+const int custom_listener_indent = 2;
+const int custom_filter_indent = 8;
+const int custom_route_indent = 14;
+
+const int fake_remote_response_indent = 14;
+const char* fake_remote_cluster_insert = "  - *fake_remote_cluster\n";
+const char* fake_remote_listener_insert = "  - *fake_remote_listener\n";
+const char* fake_remote_route_insert = "              - *fake_remote_route\n";
+
+const std::string config_header = R"(
```

Comment:
Is there any way for us to verify that the loaded config to the engine is the same? Maybe diff between startup logs, which print out the config being loaded?

Reply:
Just as a sanity check

Reply:
I'm not sure I completely understand the suggestion - but the config would be very obviously broken if the aliases weren't working.

Reply:
I am less concerned about a full break, and more about more subtle problems that might be cause by bad indentation or copy paste. 

If you run the app in debug it will print out the final bootstrap it is loading. So we can diff main and this PR to make sure that the config is 1:1.